scripts/cargo_sort/cargo_sort.py

```diff
@@ -66,7 +66,10 @@ def __init__(self, line):
             self.unknown_lines_end = []
         
         def add_node(self, node):
-            if not node.name in internal_crates_dict:
+            # we only want to add "internal crates" if we are in a "dependencies" section,
+            # otherwise we treat everything as external crates, so we don't have different
+            # groups for external and internal crates, but we still sort them.
+            if ('dependencies' not in self.line) or (not node.name in internal_crates_dict):
```

Comment:
Just making sure, that also accounts for dev-dependencies and build-dependencies, right?